helm/chart/router/templates/service.yaml

```diff
@@ -45,3 +45,7 @@ spec:
     {{- end }}
   selector:
     {{- include "router.selectorLabels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
```

Comment:
I don't see annotations as part of the service spec schema:
https://github.com/instrumenta/kubernetes-json-schema/blob/master/v1.6.4-local/servicespec.json

The annotations are already in the metadata. I think these will be ignored?

Reply:
I made the change direct to this branch. Approved, so just waiting if you agree.